src/web/app/pages-admin/admin-home-page/admin-home-page.component.ts

```diff
@@ -133,4 +139,25 @@ export class AdminHomePageComponent {
     }
   }
 
+  /**
+   * Opens a modal containing more information about how to handle common cases
+   * when trying to add an account request that already exists.
+   */
+  showAlreadyExistsModal(): void {
+    const modalContent: string = `
+      If the instructor is still unregistered, you can:
+      <ul>
+        <li>Remind the user of their account registration link
+      </ul>
+      If the instructor has already registered, you can either:
+      <ul>
+        <li>Reset the account request OR
+        <li>Reset the google account and resend the course join link OR
+        <li>Run the google ID migration script if there are many instructors/students tied to the account
+      </ul>
```

Comment:
This is a given knowledge/SOP, so adding as information here has minimum value add.

What should be done here is adding more info to better indicate which action can/should be done, e.g. which google ID is used for the said account, how many instructor/student entities are tied with the said google ID, etc.

Reply:
Two quick questions if you don't mind,

1. Is there anyway to get `googleID` from just email, name and institute?
2. Seems like the information we want to display is already available on Account Details Page and Admin Search Page. Would it be sufficient to link to either page?

Reply:
1. Not that I'm aware of. Need to add an endpoint for it, if necessary. Can only use email because (1) institute is not part of account (anymore), and (2) the name may not be the exact same (sometimes a user requests with his/her salutation in it, but not the subsequent time, or vice versa).
2. Linking to account details page makes more sense. Not much point of linking to search page because (1) we already know what needs to be "searched", and (2) accounts are not searchable, i.e. if we rely on instructor's attached google IDs, we will miss orphan accounts.

Reply:
Ok, will try this out. Just to confirm, for 1, multiple accounts might have the same email right?

Reply:
Yes, although the probability is low, it is not zero.